sw/airborne/modules/telemetry/telemetry_intermcu_fbw.c

```diff
@@ -29,11 +29,18 @@
 #include "pprzlink/pprz_transport.h"
 #include "pprzlink/intermcu_msg.h"
 #include "subsystems/datalink/telemetry.h"
+#define MSG_SIZE 128
+
+typedef enum {FBW_MODE_MANUAL = 0, FBW_MODE_AUTO = 1, FBW_MODE_FAILSAFE = 2} fbw_mode_enum;
```

Comment:
It seems this definition already exist in an other c file, and as defines in a header. Maybe it should be unified.